lib/tasks/plugin_test.rake

```diff
@@ -0,0 +1,99 @@
+require File.expand_path('../../config/environment', File.dirname(__FILE__))
```

Comment:
Lucky catch, I was just trying to precompile a plugin's assets and still had this branch checked out.  This causes problems with the Rails environment always being initialised.  Is it needed, or can we rely solely on the [:environment] dependencies below?

e.g.

<pre>
$ rake plugin:assets:precompile[foreman_setup] --trace
[.. debug logs of env startup ..]
** Invoke plugin:assets:precompile (first_time)
** Execute plugin:assets:precompile
rake aborted!
Application has been already initialized.
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/railties-3.2.21/lib/rails/application.rb:135:in `initialize!'
/home/dcleal/code/foreman/foreman/lib/tasks/plugin_assets.rake:84:in `block in <top (required)>'
</pre>